Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -18,19 +18,52 @@
 #import "ABContactInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
+@interface __ABContactOperation : NSObject
+
+@property BOOL shouldDelete;
+@property WACContact* contact;
+
+- (id)initWithContact:(WACContact*)contact shouldDelete:(BOOL)shouldDelete;
+
+@end
+
+@implementation __ABContactOperation
+
+- (id)initWithContact:(WACContact*)contact shouldDelete:(BOOL)shouldDelete {
+    if (self = [super init]) {
+        self.shouldDelete = shouldDelete;
+        self.contact = contact;
+    }
+
+    return self;
+}
+
+@end
+
 @interface _ABAddressBookManager ()
 
+- (void)_setError:(CFErrorRef*)error message:(NSString*)message;
```

Comment:
Since it's a private helper function you can make it a normal function call instead of an objective c message. This will avoid a dynamic look up.

---
In reply to: [76687800](https://github.com/Microsoft/WinObjC/pull/935#discussion_r76687800) [](ancestors = 76687800,76674447)